### PR TITLE
Release 4.23.0

### PR DESCRIPTION
**Pull request by FhanOmise**
#### 🚀 Enhancements
- Added Boost, DuitNow QR, DuitNow Online Banking/Wallets, Maybank QRPay and ShopeePay payment methods. Update Touch 'n Go to support non Alipay+. (PR [#287](https://github.com/omise/omise-woocommerce/pull/287))

### CODE REVIEW

**Comment by som-m** [comment]
@FhanOmise Could you also please help add the missing `)` in the previous release? 🙏🏻

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+### [v4.23.0 _(Jul 25, 2022)_](https://github.com/omise/omise-woocommerce/releases/tag/v4.23.0)
```
Comment (som-m) [inline CHANGELOG.md:3]: Should there be a newline here?

Comment (som-m) [inline CHANGELOG.md:3]:
```suggestion
### [v4.23.0 _(Jul 25, 2022)_](https://github.com/omise/omise-woocommerce/releases/tag/v4.23.0)

```

Comment (FhanOmise) [reply CHANGELOG.md:3]: Updated here [c53ed3e](https://github.com/omise/omise-woocommerce/pull/289/commits/c53ed3e5cade4925422b9ef773ef50a8aa44e028) ka

**Review by kan98 — APPROVED** [review]

**Review by som-m — APPROVED** [review]

**Review by tookaa2 — APPROVED** [review]

**Review by hpariwat — APPROVED** [review]

**Review by aashishgurung — APPROVED** [review]
LGTM

**Review by new4762 — APPROVED** [review]

**Pull request merged**